examples/hooks/components/layout/Tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { cx } from '../../cx';
+
+import './Tabs.css';
+
+export type TabProps = {
+  title: string;
+};
+
+export const Tabs = ({ children }) => {
+  const [currentTab, setCurrentTab] = useState(0);
+
+  return (
+    <div className="ais-Tabs">
+      <ul className="ais-Tabs-header">
+        {React.Children.map<React.ReactChild, React.ReactElement<TabProps>>(
+          children,
+          (child, index) => (
+            <li
+              className={cx(
+                'ais-Tabs-title',
+                currentTab === index && 'ais-Tabs-title--active'
+              )}
+              key={index}
+              onClick={() => setCurrentTab(index)}
+            >
+              {child.props.title}
+            </li>
+          )
+        )}
+      </ul>
+      <ul className="ais-Tabs-list">
+        {React.Children.map(children, (child, index) => (
+          <li
+            className={cx(
+              'ais-Tabs-item',
+              currentTab === index && 'ais-Tabs-item--active'
+            )}
+            key={index}
+          >
+            {child}
+          </li>
+        ))}
+      </ul>
```

Comment:
See [here](https://github.com/algolia/react-instantsearch/pull/3224/files?authenticity_token=FQqTEgZ0AbCbLx1FVOcRzLs9DhcVP17M8uaC7vwEsR3EkIgq%2FW3hLvsHlBSrIvdN4dvHpmZB1%2BYsfSZ0QoPCxQ%3D%3D&file-filters%5B%5D=.css&file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx#r763389925) we should be fine with just `<div>`s and the right accessibility attributes.